code/game/turfs/open.dm

```diff
@@ -145,7 +145,7 @@
 			C.slipping = TRUE
```

Comment:
i still can't reproduce space wind pushing during slips with either of the configurations

can someone try and reproduce it